resources/views/revisions-page.blade.php

```diff
@@ -74,7 +74,7 @@ class="divide-y divide-gray-200 dark:divide-white/10"
                     >
                         @foreach ($this->revisionsList as $version)
                             <li
-                                wire:click="showVersion({{ $version->id }})"
+                                wire:click="showVersion('{{ $version->id }}')"
```

Comment:
```suggestion
                                wire:click="showVersion(@js($version->getKey()))"
```

Can we try this please.
I should have done this initially. Good catch.